driver.go

```diff
@@ -192,38 +192,9 @@ func (l *LinstorDriver) newParams(name string, options map[string]string) (*Lins
 	return params, nil
 }
 
-func (l *LinstorDriver) Create(req *volume.CreateRequest) (err error) {
-	params, err := l.newParams(req.Name, req.Options)
-	if err != nil {
-		return err
-	}
-	c, err := l.newClient()
-	if err != nil {
-		return err
-	}
+func (l *LinstorDriver) resourcesCreate(c *client.Client, req *volume.CreateRequest, params *LinstorParams) error {
 	ctx := context.Background()
```

Comment:
I think it would be cleaner to also pass in the `ctx` as parameter. Does not really matter here, but there is already a context (that is background and never canceled anyways) that should be reused here.